code/auth-service/src/controllers/authorisation.js

```diff
@@ -29,10 +29,11 @@ function getPermissionsForUser(request, response) {
   return getRoles(userId)
     .then((roles) => {
       const permissions = getPermissions(roles);
+      logger.info(`getPermissionsForUser userId ${userId} permissions ${permissions}`);
       return response.json({ permissions });
     })
     .catch((err) => {
-      logger.error('Responding with status code 500');
+      logger.error(`Responding with status code 500 because of ${err}`);
```

Comment:
Not sure I like the error message, I'm aware you didn't write all of it...
I'd prefer more context about what went wrong
```
Error retrieving roles for user ${userId}: ${err}
```